convert2rhel/rhelvariant.py

```diff
@@ -18,6 +18,8 @@
 import os
 import re
 import logging
+
+from convert2rhel.toolopts import tool_opts
```

Comment:
Have you verified that this does not cause a circular import loop because of https://github.com/oamg/convert2rhel/blob/master/convert2rhel/toolopts.py#L20?

Reply:
I'll take a look. Thank you for pointing.